apps/site-projects/src/components/modules/Resources/YourProjects/YourProjects.js

```diff
@@ -17,25 +13,45 @@ function YourProjects({projects}) {
                   gap: '1.5rem',
                 }}
               >
-                {projects.map((project) => {
-                  return (
-                    <ProjectContainer key={project.id}>
-                      <Card
-                        isLinkingInside
-                        style={{ margin: 0, width: '100%', height: '100%' }}
-                        cardData={{
-                          id: project.id,
-                          title: project.title,
-                          secondaryText: `Commitment level: ${project.commitmentLevel}`,
-                          tags: project.interests?.map(({ interest }) => interest),
-                          description: project.catchPhrase,
-                          href: project.slug,
-                          imageSrc: project?.heroImage?.url,
-                        }}
-                      />
-                    </ProjectContainer>
-                  );
-                })}
+                {userProjects ? (
+                  userProjects.map((project) => {
+                    return (
+                      <ProjectContainer key={project.id}>
+                        <div style={{ position: 'relative', width: '100%', height: '100%', border: selectedCard.id == project.id ? '6px solid #3A7CA5' : 'none' }}>
+                        <button style={{

```

Comment:
also here it works nicely i can select the projects and the project title bellow changes along with the border of the card, the only thing i would add here is using **StyledComponents** instead of adding styles in the **style prop** to seperate the css from the code and make it lighter, but otherwise overall good work !

Reply:
thanks for the review, the changes were already made